Makefile

```diff
@@ -48,6 +48,17 @@ KUBERLR_DIRS = cmd pkg internal
 # go source files, ignore vendor directory
 KUBERLR_SRCS = $(shell find $(KUBERLR_DIRS) -type f -name '*.go')
 
+# Define target platforms, image builder and the fully qualified image name.
+TARGET_PLATFORMS ?= linux/amd64,linux/arm64,linux/s390x
+
+RUNNER := docker
+IMAGE_BUILDER := $(RUNNER) buildx
+MACHINE := rancher
```

Comment:
Can we use something more generic like "kwctl" and allow the user to overload that if needed?